tutorials/advanced_features/Serialization.ipynb

```diff
@@ -9,10 +9,11 @@
   },
```

Comment:
<p>It's not only for the human-readability, there are better arguments but they're not really relevant to get into here. What's most important is to stress that the abstract representation should always be the preferred way to serialize a sequence (even if readability is not relevant).</p><p>Also, you are not showing only 180 characters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>